mirage/unikernel.ml

```diff
@@ -47,6 +41,56 @@ module K = struct
         [ "dns-upstream" ]
     in
     Mirage_runtime.register_arg Arg.(value & opt (some string) None doc)
+
+  (* DNSmasq configuration options *)
+  (* TODO support multiple dhcp-range statements *)
+  let dhcp_range =
+    let doc = Arg.info ~doc:"Enable DHCP server." [ "dhcp-range" ] in
+    Mirage_runtime.register_arg
+      Arg.(value & opt Config_parser.(some dhcp_range_c) None doc)
+
+  let ignored = "DNSMASQ COMPATIBILITY OPTIONS (IGNORED)"
+
+  let interface =
+    let doc =
+      Arg.info ~docs:ignored ~doc:"Interface to listen on." [ "interface" ]
+    in
+    Mirage_runtime.register_arg
+      Arg.(value & opt Config_parser.(some (ignore_c "interface")) None doc)
```

Comment:
Annoyingly this collides with the mirage unix argument --interface /o\

Reply:
ok, I'm not sure what to do...

Reply:
I guess we can figure out a way to rename the unix interface option